docs/users_guide/ocean/tasks/seamount.md

```diff
@@ -0,0 +1,170 @@
+(ocean-seamount)=
+
+# seamount
+
+The ``ocean/seamount`` test is a standard sigma coordinate test problem, which is documented in 
+[Beckmann and Haidvogel (1993)](https://journals.ametsoc.org/view/journals/phoc/23/8/1520-0485_1993_023_1736_nsofaa_2_0_co_2.xml), [Haidvogel et al. (1993)](https://journals.ametsoc.org/view/journals/phoc/23/11/1520-0485_1993_023_2373_nsofaa_2_0_co_2.xml) and [Shchepetkin and McWilliams (2003)](https://agupubs.onlinelibrary.wiley.com/doi/10.1029/2001JC001047).
+This case tests the error due to pressure gradients in tilted layers. 
+
+## supported models
+
+These tasks support only MPAS-Ocean.
+
+(ocean-seamount-default)=
+
+## default task
+
+### description
+
+The test case begins with a zero velocity field and is unforced, so the exact solution is to remain motionless. 
+The seamount rises from a flat sea floor in the center of the domain. 
+In a pure z-level vertical coordinate without partial bottom cells (`partial_cell_type = full`), the pressure gradient will remain zero and induce no flow to machine precision. When any layer tilting is added, including from partial bottom cells, some flow is introduced by the pressure gradient error. This is fundamentally because the pressure must be extrapolated vertically at cell centers to the mid-depth of the edge. The default setting is the sigma coordinate. These are the images produced in the `viz` folder.
+
+```{image} images/seamount_velocity_max_t.png
+:align: center
+:width: 700 px
+```
+
+```{image} images/seamount_final_kineticEnergyCell_section.png
+:align: center
+:width: 700 px
+```
+
+```{image} images/seamount_final_normalVelocity.png
+:align: center
+:width: 400 px
+```
+
+
+### mesh
+
+The domain is planar and periodic on the zonal boundaries and solid on the
```

Comment:
My suggestion, which you implemented, was doubly periodic. Which is compatible with the Schpetkin and McWilliams paper?